src/coprocessor_v2/config.rs

```diff
@@ -0,0 +1,16 @@
+// Copyright 2021 TiKV Project Authors. Licensed under Apache-2.0.
+
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
+#[serde(default)]
+#[serde(rename_all = "kebab-case")]
+pub struct Config {
+    pub coprocessor_plugin_directory: String,
+}
+
+impl Default for Config {
+    fn default() -> Config {
+        Config {
+            coprocessor_plugin_directory: "coprocessors".to_string(),
```

Comment:
I don't think it's a good idea to use `coprocessors` as the default config. It's better to be `None`. From my perspective, I would be surprised if there's a `coprocessors` directory popping out after running `tikv-server`, when I don't explicitly enable this feature.

Reply:
Ah yes, good idea! I didn't think of that. Will implement it :)

Reply:
Done :blush: